### PR TITLE
Hack together a dts for the olimex teres-i

### PR DESCRIPTION
**Pull request by Asara**
Lifted the dts from the mainline kernel and added some changes from https://github.com/icenowy/u-boot/tree/teres-i-mainline

Keyboard currently does not work during u-boot.  This is probably not an acceptable PR in its current form, but I just wanted to get things started on getting mainline support for the Teres-I.

### CODE REVIEW

File: arch/arm/dts/sun50i-a64-teres-i.dts [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+			gpios = <&pio 7 1 GPIO_ACTIVE_HIGH>, /* sda - PL9 */
+				<&pio 7 0 GPIO_ACTIVE_HIGH>; /* scl - PL8 */
+			i2c-gpio,sda-open-drain;
+			i2c-gpio,scl-open-drain;
+			i2c-gpio,delay-us = <2>;	/* ~100 kHz */
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "okay";
+
+			anx6345: edp-bridge@38 {
```
Comment (anarsoul) [inline arch/arm/dts/sun50i-a64-teres-i.dts]: It's not in mainline linux yet, so you should the *-u-boot.dtsi, just as I did for pinebook

File: arch/arm/dts/sun50i-a64-teres-i.dts [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+				compatible = "analogix,anx6345";
+				reg = <0x38>;
+				sleep-gpios = <&pio 7 8 GPIO_ACTIVE_LOW>; /* PH8 dummy */
```
Comment (anarsoul) [inline arch/arm/dts/sun50i-a64-teres-i.dts]: No need for dummy GPIOs.

File: configs/teres_i_defconfig [hunk 3]
```diff
@@ -0,0 +1,35 @@
+CONFIG_ARM=y
```
Comment (anarsoul) [inline configs/teres_i_defconfig]: Do 'make savedefconfig' and use produced 'defconfig'. Yours has a lot of garbage, e.g. there's no AXP803 driver in u-boot, so CONFIG_AXP_DLDO2_VOLT has no effect. See pinebook defconfig for a reference.

**Comment by bufferoverflow** [comment]
u-boot community does not handle PRs on GitHub, please read https://github.com/u-boot/u-boot/blob/master/README

**Pull request closed without merging**